htsohm/simulation/gas_adsorption.py

```diff
@@ -167,7 +167,7 @@ def run(run_id, material_id, helium_void_fraction=None):
                     output_file = os.path.join(output_subdir, file)
             print('OUTPUT FILE:\t%s' % output_file)
             results = parse_output(output_file)
-#            shutil.rmtree(output_dir, ignore_errors=True)
+            shutil.rmtree(output_dir, ignore_errors=True)
```

Comment:
So it looks like you're debugging across commits. This is the sort of stuff that should get squashed out.